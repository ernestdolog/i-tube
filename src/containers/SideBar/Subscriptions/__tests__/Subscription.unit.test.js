import React from 'react';
import {shallow} from 'enzyme';
import Subscription from '../Subscription';

test('renders Subscription', () => {
  const wrapper = shallow(
    <Subscription />
  );
  expect(wrapper).toMatchSnapshot();
});

test('renders Subscription with broadcasting', () => {
    const wrapper = shallow(
        <Subscription label='MusicChannel' broadcasting/>
    );
    expect(wrapper).toMatchSnapshot();
  });

  test('renders Subscription with amountNewVideos', () => {
    const wrapper = shallow(
        <Subscription label='Coursea' amountNewVideos={10}/>
    );
    expect(wrapper).toMatchSnapshot();
  });