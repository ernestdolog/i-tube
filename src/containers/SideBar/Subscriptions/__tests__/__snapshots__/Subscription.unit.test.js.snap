// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Subscription 1`] = `
<MenuItem>
  <div
    className="subscription"
  >
    <div>
      <Image
        as="img"
        avatar={true}
        src="http://via.placeholder.com/28x28"
        ui={true}
      />
      <span />
    </div>
  </div>
</MenuItem>
`;

exports[`renders Subscription with amountNewVideos 1`] = `
<MenuItem>
  <div
    className="subscription"
  >
    <div>
      <Image
        as="img"
        avatar={true}
        src="http://via.placeholder.com/28x28"
        ui={true}
      />
      <span>
        Coursea
      </span>
    </div>
    <span
      className="new-videos-count"
    >
      10
    </span>
  </div>
</MenuItem>
`;

exports[`renders Subscription with broadcasting 1`] = `
<MenuItem>
  <div
    className="subscription"
  >
    <div>
      <Image
        as="img"
        avatar={true}
        src="http://via.placeholder.com/28x28"
        ui={true}
      />
      <span>
        MusicChannel
      </span>
    </div>
    <Icon
      as="i"
      name="signal"
    />
  </div>
</MenuItem>
`;
